check_repo.py

```diff
@@ -33,15 +32,9 @@ def check_repo():
         if addon_folder[0] != '.':
             addon_path = os.path.join(repo_path, addon_folder)
             error_counter = check_addon.start(error_counter, addon_path, config)
-
-    if error_counter["problems"] > 0:
-        colorPrint("We found %s problems and %s warnings, please check the logfile." % (
-            error_counter["problems"], error_counter["warnings"]), "31")
-        sys.exit(1)
-    elif error_counter["warnings"] > 0:
-        # If we only found warnings, don't mark the build as broken
-        colorPrint("We found %s problems and %s warnings, please check the logfile." % (
-            error_counter["problems"], error_counter["warnings"]), "35")
```

Comment:
this should be still kept
- without `sys.exit(1)` PRs won't be marked as failed
- travis log should still output an end result

Reply:
And as an info, this is designed to be able to run on PRs/repos via travis and on your local machine, without modification. So it might make sense to make the github comments opt-in via the `.tests-config.json`?